easybuild/tools/filetools.py

```diff
@@ -296,7 +408,7 @@ def find_easyconfigs(path, ignore_dirs=None):
                 continue
 
             spec = os.path.join(dirpath, f)
-            _log.debug("Found easyconfig %s" % spec)
+            _log.debug("Found easyconfig %s", spec)
```

Comment:
@riccardomurri you can try my suggestion above on https://github.com/hpcugent/easybuild-framework/blob/master/easybuild/tools/build_log.py#L109 for your #1008